examples/rig-kafka-pub-sub-example/example/src/main/java/com/rig/example/model/EventPayload.java

```diff
@@ -0,0 +1,16 @@
+package com.rig.example.model;
+
+import lombok.Data;
+
+@Data
+public class EventPayload {
+    public String payload;
+    public Integer number;
+
+    public EventPayload transform() {
```

Comment:
I would probably rather do this transform in the consumePolicyOwner method, as it is more like "business related operation" and there is no need to have it defined on the model